packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -531,25 +539,6 @@ export default class NewVm extends BaseComponent {
     poolId => sr =>
       (poolId == null || poolId === sr.$pool) && sr.SR_type === 'iso'
   )
-  _getIpPoolPredicate = createSelector(
-    () => !!this.props.pool,
-    () => {
-      const { resourceSet } = this.props
-      return resourceSet && resourceSet.ipPools
-    },
-    () => this.props.vif,
-    (pool, ipPools, vif) => ipPool => {
-      if (!ipPool) {
-        return false
-      }
-      return (
-        pool ||
-        (ipPools &&
-          includes(ipPools, ipPool.id) &&
-          find(ipPool.networks, ipPoolNetwork => ipPoolNetwork === vif.network))
-      )
-    }
-  )
```

Comment:
Why is this removed?

Reply:
It wasn't used. I guess it was before the `SelectResourceSetIp` existed.
But I just noticed that the `ipPoolPredicate` prop is not even used either. That's weird, let me check that!